frontend/app/.server/domain/multi-channel/services/case-api-service.ts

```diff
@@ -0,0 +1,29 @@
+import { mockCaseApi } from '~/.server/domain/multi-channel/mocks/case-api-service.mock';
+import { serverEnvironment } from '~/.server/environment';
+import type { InPersonSinApplication } from '~/.server/shared/services/sin-application-service';
+import { AppError } from '~/errors/app-error';
+import { ErrorCodes } from '~/errors/error-codes';
+
+export type PersonSinCase = InPersonSinApplication & { caseId: string };
```

Comment:
I get what you're doing here, but I don't think this should be directly coupled with the application type. The application type is for applying for a SIN.. the data you're returning from this service represents data that has already been processed. 

There will certainly be overlap, but it's not the same.